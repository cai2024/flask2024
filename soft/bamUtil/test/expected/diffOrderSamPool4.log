WARNING: Matching records may incorrectly be reported as mismatches due to running out of available records 1 time.
Try increasing --recPoolSize from 4 or setting it to -1 (unlimited).
