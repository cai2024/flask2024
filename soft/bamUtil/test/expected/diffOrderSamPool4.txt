1
>	a1	1:70	3S1M1S	ACGTN	;46>>	OP:i:75	MD:Z:30A0C5
18:462+29M5I3M:F:295
<	a1	1:78
>	a1	1:74
2
>	a1	1:72	3S1M1S	ACGTN	;47>>	OP:i:75	MD:Z:30A0C5
ABC
>	cd	*:0	*	*	*
DEF
>	cd	*:0	*	*	*
